haystack/nodes/reader/base.py

```diff
@@ -7,10 +7,25 @@
 from functools import wraps
 from time import perf_counter
 
-from haystack.schema import Document, Answer, Span
+from haystack.schema import Document, Answer, Span, MultiLabel
 from haystack.nodes.base import BaseComponent
 
 
+def add_doc_meta_data_to_answer(documents: List[Document], answer):
```

Comment:
Is this method used outside of Reader? If yes, great. If not maybe rather a class method?